Documentation/Settings.md

```diff
@@ -115,27 +115,33 @@ When the device powers up, should it enter into a special mode. These settings s
 
 On device help text:
 
-O=off | S=soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved
+O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby at no heat heat until moved
 
 ### Setting: Cooldown flashing
 
 If the idle screen should blink the tip temperature for attention while the tip is over 50°C. Intended as a 'tip is still hot' warning.
 
 On device help text:
 
-Flash the temperature reading after heating was halted while the tip is still hot
+Flash the temperature reading, if tip is still hot, after halting heat
```

Comment:
```suggestion
Flash the temperature reading after heating was halted while the tip is still hot
```

What is wrong about the current sentence?
It was a really tedious job to make all the 'scrolling texts' as short as possible while preserving the clarity of the content.